Python/TSV to CSV/TSV_to_CSV.py

```diff
@@ -0,0 +1,14 @@
+file_path = input("Enter the tsv file path : ")
```

Comment:
Instead of Dynamically Passing Inputs, can you have Argument Parsers to pass the File Name?

Reply:
can you please explain me more about this!

Reply:
See [this](https://docs.python.org/3/library/argparse.html)

Reply:
@Siddhant-K-code Let me know if you are able to implement it. We can still go without it.

Reply:
I think i am unable to do this by that method, can we continue with this only.